common/nym-lp/src/message.rs

```diff
@@ -149,9 +242,22 @@ impl LpMessage {
             LpMessage::Busy => 0,
             LpMessage::Handshake(payload) => payload.0.len(),
             LpMessage::EncryptedData(payload) => payload.0.len(),
-            LpMessage::ClientHello(_) => 97, // 32 bytes x25519 key + 32 bytes ed25519 key + 32 bytes salt + 1 byte bincode overhead
+            // 4 bytes receiver_index + 32 bytes x25519 key + 32 bytes ed25519 key + 32 bytes salt + bincode overhead
```

Comment:
I really don't like the fact we have some magic serialiser specific overhead. what if the  implementation changes? the client hello data is simple enough to get rid of it (we could just concat all fields since they have constant lengths)